backend/settings/default.py

```diff
@@ -29,7 +29,8 @@
 
 # TASK MANAGER SETTINGS
 KEEP_RUNNING_TASK_MANAGER: bool = True
-MAX_TIMEOUT_SECONDS: int = 5 * 60  # 5 minutes = 300 seconds
+MAX_TIMEOUT_SECONDS: int = int(os.getenv("MAX_TIMEOUT_SECONDS") or 5 * 60)  # 5 minutes = 300 seconds
+NUMBER_OF_ACTIVE_TASK_PER_MANAGER = int(os.getenv("NUMBER_OF_ACTIVE_TASK_PER_MANAGER"))
```

Comment:
in our first impl, we need to have a limit on the number of active tasks per manager.. have you already implemented this?

Reply:
no yet